translate/convert/test_po2prop.py

```diff
@@ -62,6 +62,15 @@ def test_space_preservation(self):
         print(propfile)
         assert propfile == propexpected
 
+    def test_no_value(self):
+        """check that we can handle keys without value"""
+        posource = '''#: KEY\nmsgctxt "KEY"\nmsgid ""\nmsgstr ""\n'''
+        proptemplate = '''KEY\n'''
+        propexpected = '''KEY = \n'''
```

Comment:
Please check that output is the same as the template, and make the test xfail. I will be landing the needed fix in `po2prop` as it seems unfair to load you with the burden of fixing roundtrip.